HISTORY.rst

```diff
@@ -20,6 +20,8 @@ Release History
 
 **IoT DPS updates**
 
+* Added RBAC support with --auth-type login.
+
```

Comment:
We might want to add something about an API-version update and some of the new added parameters (`--device-info`, etc)

Reply:
done - could not find official documentation for rbac for dps yet.